docs/source/deploying/workers.rst

```diff
@@ -0,0 +1,58 @@
+Task Workers
+=============
+
+Quetz offers 3 types of backends for task workers. Each of them is explained below.
+
+Thread
+-----------
+
+Thread workers process tasks in a separate thread. This functionality is in-built into FastAPI using
+``BackgroundTasks`` (https://fastapi.tiangolo.com/tutorial/background-tasks/).
+
+
+Subprocess
+-----------
+
+Subprocess workers start in a separate process and are implemented through ``ProcessPoolExecutor`` of the
+``concurrent.futures`` module. Once again, this is shipped as a part of Quetz.
+
+
+Redis
+-----------
+
+For advanced use-cases, Quetz also offers the ability to use ``redis-queue`` to manage jobs and run them on
+multiple processes or even multiple servers.
+
+To use this backend, one needs to setup ``redis`` and ``redis-queue``.
+
+Setting up ``redis``
+^^^^^^^^^^^^^^^^^^^^
+
+Make sure that ``redis`` is installed. There are multiple ways to do this. One can compile it from source (recommended),
+use a package manager for your distribution (such as ``brew`` for MacOS) and even use a Docker image.
```

Comment:
```suggestion
use a package manager for your distribution (such as ``brew`` for MacOS, ``apt-get`` for Debian/Ubuntu) or use a Docker `redis image`_ (`docker pull redis` if you have docker installed).

.. _redis image: https://hub.docker.com/_/redis/
```

i'm adding docker instructions because we already have instructions for postgres, so it's good for consistency

Reply:
Thanks